.github/workflows/pull_request.yml

```diff
@@ -32,15 +31,14 @@ jobs:
   python-sdk:
     uses: ./.github/workflows/python_tests.yml
     needs: build-template
-    if: always() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled')
     secrets:
       E2B_API_KEY: ${{ secrets.E2B_API_KEY }}
     with:
       E2B_DOMAIN: ${{ vars.E2B_DOMAIN }}
       E2B_TESTS_TEMPLATE: ${{ needs.build-template.outputs.template_id }}
   cleanup-build-template:
     uses: ./.github/workflows/cleanup_build_template.yml
-    needs: [build-template, js-sdk, python-sdk]
+    needs: [build-template]
```

Comment:
you need `js-sdk, python-sdk`, otherwise it will delete it sooner

Reply:
aaaah, yes